corehq/apps/users/views/__init__.py

```diff
@@ -949,11 +949,13 @@ def delete_user_role(request, domain):
     if user_count:
         return JsonResponse({
             "message": ngettext(
-                "Unable to delete role '{role}'. It has one user still assigned to it. "
+                "Unable to delete role '{role}'. "
+                "It has one user and pending invitations still assigned to it. "
```

Comment:
You're also picking up expired invitations, which aren't really pending. Your logic is right, since expired invitations can be [reactivated](https://github.com/dimagi/commcare-hq/blob/2e8d73844f59eacf5f72bd9bf5b9b6a0e5c8f445/corehq/apps/users/views/web.py#L233-L243). I'd just remove the word "pending" from the text.